javascript-loops/main.js

```diff
@@ -0,0 +1,68 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 1;
+  for (var i = 0; i < 10; i++) {
+    numbers.push(currentNumber);
+    currentNumber++;
+  }
```

Comment:
This should be a `while` loop

Reply:
Hi Cody, just to clarify when is it best to use a while loop ?